include/pybind11/detail/class.h

```diff
@@ -24,6 +24,14 @@ PYBIND11_NAMESPACE_BEGIN(detail)
 #  define PYBIND11_SET_OLDPY_QUALNAME(obj, nameobj) setattr((PyObject *) obj, "__qualname__", nameobj)
 #endif
 
+inline std::string get_tp_name(PyTypeObject *type) {
```

Comment:
`get_fq_type_name` or even `get_fully_qualified_type_name`?
Somehow conveying that this function is meant to produce the fully qualified name seems important to me.

Reply:
Yes, good point!

Reply:
Done! Thanks for the suggestion.